apps/web/src/services/dashboards/shared/dashboard-widget-input-form/dashboard-widget-options-form/DashboardWidgetOptionDropdown.vue

```diff
@@ -73,39 +75,52 @@ watch(() => props.inherit, () => {
             <template #label-extra>
                 <div class="inherit-toggle-button-wrapper">
                     <span class="text"
-                          :class="{inherit: !props.inherit}"
+                          :class="{inherit: props.inheritanceMode !== 'NONE'}"
                     >{{ $t('DASHBOARDS.CUSTOMIZE.ADD_WIDGET.INHERIT') }}</span>
                     <p-toggle-button :value="props.inherit"
-                                     :disabled="props.nonInheritable"
+                                     :disabled="props.inheritanceMode === 'NONE'"
                                      @change-toggle="emit('update:inherit', $event)"
                     />
                 </div>
             </template>
-            <div class="select-form-wrapper">
-                <p-select-dropdown use-fixed-menu-style
-                                   :is-filterable="!props.inherit"
-                                   :multi-selectable="props.selectionType === 'MULTI'"
-                                   :handler="props.menuHandlers"
-                                   :selected="props.selected"
-                                   :invalid="!!state.errorMessage"
-                                   @update:selected="handleUpdateSelected"
-                                   @update:visible-menu="handleUpdateVisibleMenu"
+            <div>
+                <div v-if="props.label !== 'Cost Tag' || props.label !== 'Cost Additional Info'"
```

Comment:
suggestion: How about receiving 'widget filter key(e.g. filters.provider)' via props?
I'm concerned that `label` can be customized in the widget config😅

Reply:
nitpick: please add comment like 'HACK'